setup.py

```diff
@@ -546,7 +546,7 @@ def run(self):
             "scipy": ["scipy"],
         },
         ext_modules=get_extensions(),
-        python_requires=">=3.7",
+        python_requires=">=3.7.2",
```

Comment:
As I commented on https://github.com/pytorch/vision/issues/6925#issuecomment-1306995720, I think there might be a more robust and future-proof way of handling the original failure.

I'm also wondering if it's safe for us to deviate from torch core's Python version requirements which seem to be set to 3.7.0 as well:

https://github.com/pytorch/pytorch/blob/8cb5c5543e92628782deb00dda78380076b89e66/setup.py#L214

@malfet could you please advise on this?